projects/knora-ui/src/lib/viewer/viewer.module.ts

```diff
@@ -63,7 +64,8 @@ import { KnoraDatePipe } from './pipes/knoradate.pipe';
         ListValueComponent,
         SublistValueComponent,
         GeonameValueComponent,
-        KnoraDatePipe
+        KnoraDatePipe,
```

Comment:
Did you consider exporting any of the pipes? Then they could be used in an app too.

If you add them to the `exports` array, also add them to the viewer's `index.ts`.

Reply:
good idea! I've exported them and added them to the viewer's `index.ts` in a36ae2d